cinder_service_check.py

```diff
@@ -53,6 +53,12 @@ def check(auth_ref, args):
 
     services = r.json()['services']
 
+    # We need to match against a host of X and X@lvm (or whatever backend)
+    if args.host:
+        backend = ''.join((args.host, '@'))
```

Comment:
@sigmavirus24 I don't understand your comment, what is it you dislike about the patch?